src/uu/dd/src/dd_unit_tests/mod.rs

```diff
@@ -0,0 +1,143 @@
+/* cspell:disable */
+
+use super::*;
+
+mod block_unblock_tests;
+mod conv_sync_tests;
+mod conversion_tests;
+mod sanity_tests;
+
+use std::fs;
+use std::io::prelude::*;
+use std::io::BufReader;
+
+const DEFAULT_CFO: OConvFlags = OConvFlags {
+    sparse: false,
+    excl: false,
+    nocreat: false,
+    notrunc: false,
+    fdatasync: false,
+    fsync: false,
+};
+
+const DEFAULT_IFLAGS: IFlags = IFlags {
+    cio: false,
+    direct: false,
+    directory: false,
+    dsync: false,
+    sync: false,
+    nocache: false,
+    nonblock: false,
+    noatime: false,
+    noctty: false,
+    nofollow: false,
+    nolinks: false,
+    binary: false,
+    text: false,
+    fullblock: false,
+    count_bytes: false,
+    skip_bytes: false,
+};
+
+// const DEFAULT_OFLAGS: OFlags = OFlags {
```

Comment:
Do we still need this? An alternative to adding these default consts would be to add `#[derive(Default)]` to `OConvFlags`, `IFlags` and `OFlags` directly, which will allow to use e.g. `OConvFlags::default()` which also fills all fields with `false`.

Reply:
Good suggestion. I've implemented this.